third-party-podspecs/RNSVG.podspec.json

```diff
@@ -11,7 +11,7 @@
   },
   "source": {
     "git": "https://github.com/wordpress-mobile/react-native-svg.git",
-    "tag": "v9.13.6-gb"
+    "tag": "v9.13.7-wp"
```

Comment:
Nitpick: I noticed that this is the only podspec file that points to a greater version (`9.13.6` --> `9.13.7`), however, I haven't seen other changes in that repo apart from the required ones for using RN 0.64 ([compare changes](https://github.com/wordpress-mobile/react-native-svg/compare/v9.13.6-gb...wordpress-mobile:v9.13.7-wp)).

Reply:
Because it was using `-gb` suffix before, I thought of bumping the version while changing the suffix to `-wp` to not have any confusions with having both `v9.13.6-gb` and `v9.13.6-wp` versions.